admin/app/views/commercial/commercialMenu.scala.html

```diff
@@ -19,7 +19,7 @@ <h3>Performance</h3>
                         <li><a href="@controllers.admin.routes.CommercialController.renderCommercialRadiator()">Commercial Radiator</a></li>
                         <li><a href="@controllers.admin.routes.CommercialController.renderBrowserPerformanceDashboard()">Real-user advert performance</a></li>
                         <li><a href="http://dashboards.gu-web.net/s3?list=dotcom-commercial/health/queries">Commercial Health</a></li>
-                        <li><a href="@controllers.admin.commercial.routes.TeamKPIController.renderPrebidDashboard()">Prebid vs Sonobi wrapper</a></li>
+                        <li><a href="@controllers.admin.commercial.routes.TeamKPIController.renderPrebidDashboard()">Prebid Bidder Performance</a></li>
```

Comment:
Is this commit intended to be in this PR? Given the innocuous nature I'm fine with you keeping it in here. It won't be the end of the world if this needs to be reverted.

Reply:
This is the seed for the PR!  I wanted to fix the link but I thought I'd better do something more useful at the same time.

Reply:
Ha I like it: justifying the work of the TeamCity agents.